### ISSUE TITLE
Display Hausumringe in FE

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#147]
## Description
See Epic: #145 

## Acceptance criteria
* Display Hausumringe (building borders) for marker in frontend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
area shapes can be viusalized and inner area is currently slightly transparent but can be easily adjusted

**Comment by Lucas-Nan** [comment]
there seem to be some problems with querying it from the database. db sometimes crashes. Has to be further investigated
